src/orquestra/sdk/_base/_driver/_models.py

```diff
@@ -271,6 +271,7 @@ class CreateWorkflowRunRequest(pydantic.BaseModel):
 
     workflowDefinitionID: WorkflowDefID
     resources: Resources
+    dry_run: bool
```

Comment:
**issue**: Is it supported on the Platform already? This field isn't in the [CE's API spec](https://github.com/zapatacomputing/workflow-driver/blob/5d030e5a71be088c86590ce63180e256b55b3c8e/openapi/src/schemas/CreateWorkflowRunRequest.yaml). Even if we want to add client code before the Platform is ready, the spec should be updated first.

Reply:
Its not supported - my intention is to wait for workflow-driver to support this feature before I merge this.